src/scripts/components/stories/pilot-story/pilot-story.tsx

```diff
@@ -24,24 +24,34 @@ const PilotStory: FunctionComponent = () => {
       <ParallaxProvider>
         <Globe
           progress={progress}
-          relativePosition={{top: 0, left: -100, right: 0, bottom: 0}}
+          relativePosition={{x: -30, y: 0}}
           isSpinning={true}
           isVisible={true}
-          viewTotal={5}
+          pagesTotal={5}
           globeMovement={[
             {
-              viewFrom: 1,
-              viewTo: 2,
-              directions: [GlobeMovementDirection.RIGHT]
+              pageFrom: 1,
+              pageTo: 2,
+              relativeExtend: 30,
```

Comment:
sets the extend for a movement. This means globe is moved by 30% right. Relative x position was set to -30,  so globe is centered after movement is finished on page 2.